protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -298,3 +298,17 @@ enum CoreOperatorType {
   NESTED_LOOP_JOIN = 35;
   AVRO_SUB_SCAN = 36;
 }
+
+message Func {
+  optional string name = 1;
+  repeated common.MajorType major_type = 2;
```

Comment:
It's type of in parameter. For example, toString of common.MajorType class:
"major_type {\n" +
 +        "  minor_type: VARCHAR\n" +
 +        "  mode: REQUIRED\n" +
 +        "}\n
I can update name to type.